packages/firestore/src/platform_browser/webchannel_connection.ts

```diff
@@ -172,7 +172,7 @@ export class WebChannelConnection implements Connection {
         }
       });
 
-      const jsonObj = {...request} as any;
+      const jsonObj = { ...request } as any;
```

Comment:
We try not to use `any` (and the linter won't let you) since it essentially disables all type-checking.  Try `object` instead?

Reply:
Poking at this a little more, this may be what you need:

```
import { Indexable } from '../util/misc';
...
const jsonObj = { ...request } as unknown as Indexable;
```

Reply:
Done.